assets/js/amazon-wc-checkout.js

```diff
@@ -13,6 +13,12 @@
 			}
 			return true;
 		} );
+		$( document.body ).on( 'added_to_cart removed_from_cart', function( fragments, cartHash, clickedBtn ) {
```

Comment:
If we wanna do this when fragments are updated, maybe it would be better to catch this: 
```
$( document.body ).trigger( 'wc_fragments_loaded' );
```
?

Reply:
yes, implemented on latest commit alogn with a fix for a bug where, checkoutsession params weren't fresh during checkout.